lib/classes/CLI.test.js

```diff
@@ -447,10 +447,12 @@ describe('CLI', () => {
     it('should log the version numbers', () => {
       cli.getVersionNumber();
 
-      expect(consoleLogSpy).to.have.been.calledOnce;
+      expect(consoleLogSpy).to.have.been.called.calledTwice;
```

Comment:
Won't this fail on our Node.js 6 builds?

I guess it would be best to mock `process.version` to control this. @medikoo what do you think?

Reply:
I would get rid of above line (as we test the outcome below, and how many console.log's it takes I think is implementation detail we should not really test)

Still below I would check for `Components...` only in Node.js v8+ (as you're right, I would expect it to fail in Node.js v6, and Travis will most likely confirm on that)

Reply:
Thanks guys! Done